modules/internal/BytesStringCommon.chpl

```diff
@@ -40,7 +40,7 @@ module BytesStringCommon {
   /*
      ``encodePolicy`` specifies what happens when there is escaped non-UTF8
      bytes when encoding a :record:`~String.string` into a
-     :record:`~Bytes.bytes`.
+     :mod:`~Bytes`.
```

Comment:
Normally we add `~` in `~Bytes.bytes` to prevent "Bytes" to be printed in the docs. It may be compiling this way, but I'd prefer you drop `~` in such cases

Reply:
Just pushed the changes removing `~` and also it passed the bytes tests for me